docs/user/security/authorization/index.asciidoc

```diff
@@ -95,9 +97,9 @@ image::user/security/images/create-role-dls-example.png[Create role with DLS ind
 To assign {kib} privileges to the role, click **Add {kib} privilege** in the {kib} section.
 
 [role="screenshot"]
-image::user/security/images/add-space-privileges.png[Add {kib} privileges]
+image::spaces/images/spaces-roles.png[Add {kib} privileges]
```

Comment:
We had the same exact image in two different places, so I deleted the one in the Security directory and changed this to refer to the one in the Spaces directory instead.